nexmo/__init__.py

```diff
@@ -424,7 +424,7 @@ def post(self, host, request_uri, params, header_auth=False):
             headers = dict(headers or {}, Authorization="Basic {hash}".format(hash=h))
         else:
             params = dict(params, api_key=self.api_key, api_secret=self.api_secret)
-        logger.debug("POST to %r with params %r, headers %r", uri, params)
+        logger.debug("POST to %r with params %r, headers %r", uri, params) # coming up with error of "not enough arguments for logging format string"
```

Comment:
Thank you for flagging this bug - would you like to fix it? I don't really want to merge this comment into our main branch, but I think you should find the fix pretty straightforward, and then you'll get a merged PR :)

Reply:
Ah, I'm sorry, someone else already fixed the bug in the logging statement, so I'm going to close this PR.